resteasy-spring-boot-starter/pom.xml

```diff
@@ -65,6 +65,13 @@
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-jaxrs</artifactId>
             <version>${resteasy.version}</version>
+            <!-- wse javax annotation classes from org.apache.tomcat:tomcat-annotations-api -->
```

Comment:
This, and the following exclusion too, can't be done, because you are under the assumption that Tomcat will always be present, which is not true. Spring Boot applications can use other containers, like Jetty or Undertow.